Packages/KeyAppKit/Sources/KeyAppNetworking/HttpClient.swift

```diff
@@ -32,7 +37,7 @@ final class HttpClientImpl: HttpClient {
             decoder.keyDecodingStrategy = .convertFromSnakeCase
```

Comment:
Lets abstract this into a DecodingStrategy with DefaultDecodingStrategy and JsonRpcDecodingStrategy. Each Endpoint should define its DecodingStrategy